File: JavaScript/2.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Digite o valor da aplicação inicial: ', (aplicacaoInicial) => {
    aplicacaoInicial = parseFloat(aplicacaoInicial);

    rl.question('Digite o período em anos: ', (anos) => {
        anos = parseInt(anos);

        rl.question('Digite a taxa de juros por período anual (%): ', (taxaJuros) => {
            taxaJuros = parseFloat(taxaJuros);
            const taxaDecimal = taxaJuros / 100;
            const juros = aplicacaoInicial * anos * taxaDecimal;

            console.log("O juro simples é: ", juros);
            rl.close();
        });
    });
});

File: Python/2.py
aplicacao_inicial = float(input("Digite o valor da aplicação inicial: "))
anos = float(input("Digite o período em anos: "))
taxa_juros = float(input("Digite a taxa de juros por período anual (%): "))
taxa_decimal = taxa_juros / 100
juros = aplicacao_inicial * anos * taxa_decimal
total = juros + aplicacao_inicial
print("O juro simples é:", juros, "dando um total de: ", total)